tests/io/test_checksumming_proxy.py

```diff
@@ -0,0 +1,53 @@
+import hashlib
+import unittest
+from io import IOBase, BytesIO
+
+from slicedimage.backends._base import ChecksummingContextManager
+
+
+class TestChecksummingProxy(unittest.TestCase):
+    def test_instanceof(self, test_inp="abcde"):
+        obj = BytesIO(test_inp)
+        with ChecksummingContextManager(obj, None) as fh:
+            self.assertIsInstance(fh, IOBase)
+
+    def test_read(self, test_inp="abcde"):
+        obj = BytesIO(test_inp)
+        with ChecksummingContextManager(obj, None) as fh:
```

Comment:
So basically, this context manager reads a file object and calculates a checksum which it passes to fh? 

Could this be `CalculateFileObjectChecksum` instead? That would read more clearly.

Reply:
Sure.